apps/studio/esbuild.mjs

```diff
@@ -94,98 +92,126 @@ const electronUtilityPlugin = {
   }
 }
 
-  const commonArgs = {
-    platform: 'node',
-    publicPath: '.',
-    outdir: 'dist',
-    bundle: true,
-    external: [...externals, '*.woff', '*.woff2', '*.ttf', '*.svg', '*.png'],
-    sourcemap: isWatching,
-    minify: !isWatching
+const electronPreloadPlugin = {
+  name: "electron-preload-restarter",
+  setup(build) {
+    if (!isWatching) return;
+    build.onStart(() => console.log("ESBUILD: Building Preload 🏗"));
+    build.onEnd(() => {
+      console.log("ESBUILD: Built Preload ✅");
+      if (electron) {
+        process.kill(electron.pid, 'SIGINT');
+      }
+      //start electron again
+      electron = spawn(path.join('../../node_modules/electron/dist/electron'), ['.'], { stdio: 'inherit' });
+    });
+  }
+}
+
+const env = isWatching ? '"development"' : '"production"';
+const commonArgs = {
+  platform: 'node',
+  publicPath: '.',
+  outdir: 'dist',
+  bundle: true,
+  external: [...externals, '*.woff', '*.woff2', '*.ttf', '*.svg', '*.png'],
```

Comment:
There's a similar `externals` variable in `vue.config.js` and I feel like this one isn't meant to be added here, no?

Reply:
I believe vue.config.js is actually useless at this point. That's for vuecli and we use esbuild now, I think I removed it in master and it just hasn't made its way over here yet

Reply:
Yeah I still see the file in master. That's cool we don't need it anymore!